crates/symbolicator/Cargo.toml

```diff
@@ -27,7 +27,7 @@ lazy_static = "1.4.0"
 log = { version = "0.4.13", features = ["serde"] }
 lru = "0.7.0"
 num_cpus = "1.13.0"
-minidump = "0.3"
+minidump = "0.9.4"
```

Comment:
this is a big jump and important change.  might want to watch this

Reply:
The version jump was purely cosmetic I think, as it went straight from 0.3 to 0.9.